modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/validation/ParameterTypeModelValidator.java

```diff
@@ -132,6 +146,17 @@ private void checkInvalidFieldAnnotations(NamedObject model, String typeName, Fi
           }
         }
       }
+
+      private boolean isPojoWithDefaultConstructor(Class<?> clazz) {
+        return (!clazz.isInterface() && !isAbstract(clazz.getModifiers())
+            && Stream.of(clazz.getConstructors())
+                .noneMatch(c -> c.getParameterCount() == 0));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the parentheses around the "c" parameter ([squid:S1611](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1611))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7615&issues=AWhdy-0YotIXnmsUGksQ&open=AWhdy-0YotIXnmsUGksQ)

[ISSUE_KEY]: AWhdy-0YotIXnmsUGksQ